### ISSUE TITLE
setup.py environment working when recreated but using conda root interpreter when not recreated

### ISSUE THREAD

**Issue by smarie** [tox-dev/tox-conda#26]
When executing the command `tox -e setup.py` the first time it works correctly, but the second time it does not seem to activate the environment and raises the following warning : 

```
WARNING: test command found but not installed in testenv
  cmd: C:\Miniconda3\envs\tools_py37\python.EXE
  env: C:\_dev\python_ws\_Libs_OpenSource\py-make\.tox\setup.py
Maybe you forgot to specify a dependency? See also the whitelist_externals envconfig setting.
```

Workaroun: if I use the `-r` flag it works again.

You can reproduce this bug by using the tox file in https://github.com/tqdm/py-make
(which is the project I am trying to contribute to, that's why I have to use tox :) )

It seems related to #20

**Comment by smarie** [comment]
Might also be a duplicate of #24 . If so, sorry for that

**Comment by maximlt** [comment]
I experience the exact same problem (version 0.2.0). The workaround `-r` flag works as suggested.

Another workaround is not to use *tox-conda* :(
Instead, this [article](https://fizzylogic.nl/2017/11/01/how-to-setup-tox-on-windows-with-anaconda/) recommends creating manually the required Python environments with `conda create -p C:\Python36 python=3.6`. This works well. Without this manual install, I get the error `ERROR: InterpreterNotFound: python3.6` (see https://github.com/tox-dev/tox/issues/378)

**Comment by rompom** [comment]
Two issues in one : the get_python_executable find only the base python exe, and if we use a hook to make it find the good one, then on the re-run of tox, the old one is found again as the hook is executed after the config check => we get a force recreate of the environment.

**Comment by obestwalter** [comment]
Did anyone try the patch? Shall we merge and release this? It looks pretty brittle but if it is a good workaround for now, it is better than nothing. In the long run it would be good to educate core about what is needed for a plugin like tox-conda to work without hacks like patching `_venv_lookup` (e.g. making it part of the API or something).

**Comment by smarie** [comment]
The PR seems to be fixing my original problem.

**Comment by obestwalter** [comment]
fixed with #35
